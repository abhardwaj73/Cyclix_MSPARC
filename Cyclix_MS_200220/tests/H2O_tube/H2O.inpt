CELL: 18.000000 0.571198664289053 3.000000 
FD_GRID: 72 28 12
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE


TOL_SCF: 1.00E-04
TOL_PSEUDOCHARGE: 1.00E-05
MIXING_VARIABLE: potential
MIXING_PRECOND: none
