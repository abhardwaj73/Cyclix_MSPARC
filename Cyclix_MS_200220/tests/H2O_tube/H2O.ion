#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name>
# PSEUDO_POT: <path/to/pseudopotential/file>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...

# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.

ATOM_TYPE: H                 # atom type
PSEUDO_POT: H_GGA.psp8
N_TYPE_ATOM: 2               # number of atoms of this type
COORD:                       # coordinates follows
12 0 0 
11.626949060527737 2.968847511054275 0

ATOM_TYPE: O               # atom type
PSEUDO_POT: O_GGA.psp8
N_TYPE_ATOM: 1               # number of atoms of this type
COORD:                       # coordinates follows
9.921976672293289 1.246747333852277 0
