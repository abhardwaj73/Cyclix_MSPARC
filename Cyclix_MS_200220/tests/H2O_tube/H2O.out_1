***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 16-Aug-2020  Start time: 17:56:29                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 8.000000 0.571199 3.000000 
FD_GRID: 32 28 12
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
CHEB_DEGREE: 31
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 2.00E-05
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/H2O_tube/H2O.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.250000 (Bohr)
Mesh spacing in y-direction        :  0.224399 (Bohr)
Mesh spacing in z direction        :  0.250000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/H2O_tube/H2O.out_1
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  1.75 0.15 1.75
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  2.75 0.23 2.75
Number of atoms of type 2          :  1
Estimated total memory usage       :  33.67 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.5227819187E+00        4.387E-01        0.792
2            -5.7618009104E+00        1.027E-01        0.229
3            -5.7648429465E+00        5.066E-02        0.208
4            -5.7652306634E+00        1.337E-02        0.202
5            -5.7653561548E+00        2.414E-03        0.179
6            -5.7653570454E+00        1.587E-03        0.173
7            -5.7653604886E+00        3.458E-04        0.173
8            -5.7653590250E+00        2.819E-04        0.159
9            -5.7653640491E+00        8.543E-05        0.178
Total number of SCF: 9     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7653640491E+00 (Ha/atom)
Total free energy                  : -1.7296092147E+01 (Ha)
Band structure energy              : -5.8405923137E+00 (Ha)
Exchange correlation energy        : -3.9626324514E+00 (Ha)
Self and correction energy         : -3.6121209290E+01 (Ha)
Entropy*kb*T                       : -1.2149588132E-05 (Ha)
Fermi level                        : -2.3667402362E-01 (Ha)
Average force                      :  1.7069914520E-01 (Ha/Bohr)
Maximum force                      :  2.0827389344E-01 (Ha/Bohr)
Time for force calculation         :  0.120 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2.918 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
