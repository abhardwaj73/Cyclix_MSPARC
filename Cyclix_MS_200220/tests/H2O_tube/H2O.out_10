***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 16-Aug-2020  Start time: 17:59:49                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 12.000000 0.571199 3.000000 
FD_GRID: 48 28 12
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
CHEB_DEGREE: 31
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-04
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/H2O_tube/H2O.out_10
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.250000 (Bohr)
Mesh spacing in y-direction        :  0.224399 (Bohr)
Mesh spacing in z direction        :  0.250000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/H2O_tube/H2O.out_10
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  1.50 0.13 1.50
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  1.75 0.15 1.75
Number of atoms of type 2          :  1
Estimated total memory usage       :  49.99 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.7915937668E+00        8.110E-01        1.185
2            -5.7637733955E+00        1.367E-01        0.387
3            -5.7735552397E+00        8.415E-02        0.324
4            -5.7755845145E+00        3.742E-02        0.332
5            -5.7756128273E+00        2.713E-02        0.300
6            -5.7757068177E+00        2.999E-03        0.300
7            -5.7757340089E+00        1.070E-03        0.273
8            -5.7757386014E+00        5.520E-04        0.266
9            -5.7757340949E+00        1.622E-04        0.254
10           -5.7757353634E+00        1.341E-04        0.197
11           -5.7757365419E+00        5.080E-05        0.195
Total number of SCF: 11    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7757365419E+00 (Ha/atom)
Total free energy                  : -1.7327209626E+01 (Ha)
Band structure energy              : -6.5966230869E+00 (Ha)
Exchange correlation energy        : -3.9263434061E+00 (Ha)
Self and correction energy         : -3.6121189665E+01 (Ha)
Entropy*kb*T                       : -4.2052012589E-05 (Ha)
Fermi level                        : -3.3152209182E-01 (Ha)
Average force                      :  1.8100778680E-01 (Ha/Bohr)
Maximum force                      :  2.2504840155E-01 (Ha/Bohr)
Time for force calculation         :  0.141 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  4.976 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
