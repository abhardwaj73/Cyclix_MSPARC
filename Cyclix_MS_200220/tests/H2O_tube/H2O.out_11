***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 16-Aug-2020  Start time: 17:59:58                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 14.000000 0.571199 3.000000 
FD_GRID: 56 28 12
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
CHEB_DEGREE: 31
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-04
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/H2O_tube/H2O.out_11
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.250000 (Bohr)
Mesh spacing in y-direction        :  0.224399 (Bohr)
Mesh spacing in z direction        :  0.250000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/H2O_tube/H2O.out_11
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  1.50 0.13 1.50
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  1.75 0.15 1.75
Number of atoms of type 2          :  1
Estimated total memory usage       :  58.16 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1675224546E+00        8.946E-01        1.520
2            -5.7139385033E+00        3.221E-01        0.595
3            -5.3596200968E+00        1.183E+00        0.533
4            -5.7606680827E+00        9.211E-02        0.542
5            -5.7606871390E+00        1.383E-01        0.460
6            -5.7674865974E+00        1.356E-01        0.378
7            -5.7745808664E+00        6.626E-02        0.467
8            -5.7755520140E+00        3.238E-02        0.411
9            -5.7759028529E+00        5.736E-03        0.413
10           -5.7759164043E+00        2.217E-03        0.406
11           -5.7758980167E+00        3.279E-03        0.323
12           -5.7759022227E+00        5.498E-04        0.331
13           -5.7758993991E+00        9.021E-05        0.224
Total number of SCF: 13    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7758993991E+00 (Ha/atom)
Total free energy                  : -1.7327698197E+01 (Ha)
Band structure energy              : -6.6956171379E+00 (Ha)
Exchange correlation energy        : -3.9235185658E+00 (Ha)
Self and correction energy         : -3.6121187443E+01 (Ha)
Entropy*kb*T                       : -5.0767999949E-05 (Ha)
Fermi level                        : -3.4407754103E-01 (Ha)
Average force                      :  1.8125021415E-01 (Ha/Bohr)
Maximum force                      :  2.2540468766E-01 (Ha/Bohr)
Time for force calculation         :  0.158 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  7.811 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
