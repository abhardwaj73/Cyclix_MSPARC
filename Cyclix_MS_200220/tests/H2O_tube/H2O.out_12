***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 16-Aug-2020  Start time: 18:00:08                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 16.000000 0.571199 3.000000 
FD_GRID: 64 28 12
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
CHEB_DEGREE: 31
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-04
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/H2O_tube/H2O.out_12
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.250000 (Bohr)
Mesh spacing in y-direction        :  0.224399 (Bohr)
Mesh spacing in z direction        :  0.250000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/H2O_tube/H2O.out_12
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  1.50 0.13 1.50
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  1.75 0.15 1.75
Number of atoms of type 2          :  1
Estimated total memory usage       :  66.32 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.6575683372E+00        9.356E-01        1.885
2            -5.5757498833E+00        1.023E-01        0.728
3            -5.2732662143E+00        1.795E+00        0.768
4            -5.1203617699E+00        1.810E+00        0.625
5            -5.7525030536E+00        6.183E-01        0.740
6            -5.7466550531E+00        2.294E-01        0.645
7            -5.7207318407E+00        2.250E-01        0.626
8            -5.7477756994E+00        1.738E-01        0.569
9            -5.7597591890E+00        1.970E-01        0.489
10           -5.7647285925E+00        1.608E-01        0.522
11           -5.7737464657E+00        5.062E-02        0.605
12           -5.7757024237E+00        4.782E-02        0.603
13           -5.7758045692E+00        1.260E-02        0.561
14           -5.7758309499E+00        3.916E-03        0.486
15           -5.7758586414E+00        2.282E-03        0.420
16           -5.7758456239E+00        9.146E-04        0.325
17           -5.7758458206E+00        3.804E-04        0.370
18           -5.7758410449E+00        9.433E-05        0.283
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7758410449E+00 (Ha/atom)
Total free energy                  : -1.7327523135E+01 (Ha)
Band structure energy              : -6.7591626864E+00 (Ha)
Exchange correlation energy        : -3.9224121499E+00 (Ha)
Self and correction energy         : -3.6121184943E+01 (Ha)
Entropy*kb*T                       : -5.6960903645E-05 (Ha)
Fermi level                        : -3.5193679997E-01 (Ha)
Average force                      :  1.8101541467E-01 (Ha/Bohr)
Maximum force                      :  2.2474895329E-01 (Ha/Bohr)
Time for force calculation         :  0.180 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  12.680 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
