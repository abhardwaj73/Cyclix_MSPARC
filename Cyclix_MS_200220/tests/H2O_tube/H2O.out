***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 16-Aug-2020  Start time: 17:56:09                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 6.000000 0.571199 3.000000 
FD_GRID: 24 28 12
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
CHEB_DEGREE: 31
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-04
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-04
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/H2O_tube/H2O.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.250000 (Bohr)
Mesh spacing in y-direction        :  0.224399 (Bohr)
Mesh spacing in z direction        :  0.250000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/H2O_tube/H2O.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  1.50 0.13 1.50
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  1.75 0.15 1.75
Number of atoms of type 2          :  1
Estimated total memory usage       :  25.51 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.2228519991E+00        4.896E-01        0.629
2            -5.7006731713E+00        1.145E-01        0.147
3            -5.7055669141E+00        6.451E-02        0.139
4            -5.7059669384E+00        2.144E-02        0.139
5            -5.7065069301E+00        3.207E-03        0.138
6            -5.7065142648E+00        9.260E-04        0.134
7            -5.7065122947E+00        2.495E-04        0.126
8            -5.7065128156E+00        9.850E-05        0.120
Total number of SCF: 8     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7065128156E+00 (Ha/atom)
Total free energy                  : -1.7119538447E+01 (Ha)
Band structure energy              : -4.3852364027E+00 (Ha)
Exchange correlation energy        : -4.0525619601E+00 (Ha)
Self and correction energy         : -3.6121189665E+01 (Ha)
Entropy*kb*T                       : -1.0093502875E-05 (Ha)
Fermi level                        : -5.1399609541E-02 (Ha)
Average force                      :  8.8652717982E-02 (Ha/Bohr)
Maximum force                      :  1.2798385479E-01 (Ha/Bohr)
Time for force calculation         :  0.084 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2.041 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
